x-pack/plugins/observability_solution/apm/public/components/app/metrics/serverless_metrics/serverless_summary.tsx

```diff
@@ -34,7 +34,7 @@ const CentralizedContainer = styled.div`
 
 const Border = styled.div`
   height: 55px;
-  border-right: 1px solid ${({ theme }) => theme.eui.euiColorLightShade};
+  border-right: 1px solid ${({ theme }) => theme.euiTheme.colors.lightShade};
```

Comment:
Suggestion: We can use a border width token here for `1px` (`euiTheme.border.width.thin`) and besides that I'm assuming this is a default border? Could we use the standard border color?

```
border-right: ${theme.euiTheme.border.thin};
```

Same suggestion for all other usage of a border property with a value of `1px solid ${({ theme }) => theme.euiTheme.colors.lightShade};` 🙂